### ISSUE TITLE
Doubts about lr .

### ISSUE THREAD

**Issue by weizh588** [open-mmlab/mmaction2#1577]
Hi, I see that the ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py file has an lr of 0.000125 and i would like to know if this value is the learning rate for fine tuning or for normal training? Thank you !

# optimizer
optimizer = dict(
    type='SGD', lr=0.000125, momentum=0.9,
    weight_decay=0.0001)  # this lr is used for 8 gpus

**Comment by Dai-Wenxun** [comment]
The backbone is pretrained, so this learning rate should be used for fine-tuning.
